### ISSUE TITLE
Visually hidden comma missing between brand title and episode title

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#8594]
**Describe the bug**
For the recent video episodes component, there should be a visually hidden comma between the brand title and episode title

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/afrique/bbc_afrique_tv/tv_programmes/w13xttmz
2. View the source code for the recent episodes component
3. Observe the missing visually hidden comma between the brand title and episode title

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
